README.md

```diff
@@ -1,20 +1,39 @@
 # Blocks Starer
 
-## Installation
-
-## Local Development
-
 ## Requirements
 
-### Development
-
 * Git
 * Composer
 * Node v16.13.1 or higher & NPM v8.1.2 or higher
 * NVM is recommended for managing multiple versions of node on the same workstation.
 
+## Local Development
+
+One of the goals of this starter is to allow developers to use whatever local development platform that works best for them. There are some details below for Lando and Local by Flywheel. If you are using a different environment, feel free to add it.
+
+### Lando
+
+You can use [Lando](https://lando.dev/download/) to for your local development. When starting a new project, change the name value in the `.lando.yml` file to the name of the project. Then run `lando start` to build the environment. The `local-config.php` is setup to support lando out of the box. Once the lando is running, you can follow the BE Setup instructions for the composer commands to finish the setup.
+
+### Local by Flywheel
+
+It is recommeneded to create a blank blueprint in Local by Flywheel in order to make it easier to startup a project. Select the blank blueprint, clone in the repository to the public folder and then follow the BE Setup instructions for getting started.
+
 ## Getting Started
 
+### BE Setup
+
+Run `composer run setup-project` to copy the `.env`, and `local-config` files over. Once that has completed, update the `.evn` file to include the required licenses for ACF Pro, and Gravity Forms. Once the keys are up to date, run `composer install` to pull in the required libraries.  Then run `composer setup-wordpress` to install WordPress using WP Cli. Depending on your local environment you may need to update your `local-config.php` for the local environment you are working in.
+
+``` bash
+composer setup-project
+# ... update .env file if you need ACF Pro and Gravity Forms
+composer install
+composer setup-wordpress
+```
+
+For WordPress updates, you can change the `--version=` value in the `setup-wordpress` composer script.
+
```

Comment:
Suggestion: It might be good to move this up before the Local Development section. I got caught up following the readme in-order and started up Lando before running the composer steps.

Reply:
You should run this with Lando started. If you are running Lando you need to run `lando composer [command]` in order to run the command in the container... 

With that being said, I'll add a quick explainer for both Local and Lando on making sure you run the composer commands in their respective containers.